src/parser/jobs/nin/modules/TrickAttackUsage.js

```diff
@@ -8,7 +8,7 @@ import Module from 'parser/core/Module'
 import {Suggestion, TieredSuggestion, SEVERITY} from 'parser/core/modules/Suggestions'
 
 const TA_COOLDOWN_MILLIS = ACTIONS.TRICK_ATTACK.cooldown * 1000
-const OPTIMAL_GCD_COUNT = 3 // Number of GCDs prior to the first TA in the opener
+const OPTIMAL_GCD_COUNT = 5 // Opener should be Suiton > AE combo > SE before Trick
 
```

Comment:
Seems like this is a little looser of a check that it could be.  Opening with TCJ before Trick would mess this count up (besides being bad for other reasons).

Reply:
While I don't love the implementation I did for this, I'm inclined to leave it as-is for now and revisit it later, since it's been this way since the initial implementation. Plus, 5th should be correct given the opener I was given/am using (Suiton > SE > GS > AE > SE before you weave Trick). Just checking the count instead of the exact GCDs used may result in false negatives, but I think it should be fine at least for now.